lblod/frontend-reglementaire-bijlage
# GN-4801: Bump appuniversum to v3.x.x

## Overview

GN-4801: Bump `@appuniversum/ember-appuniversum` to `3.4.1`

* Bump `@appuniversum/ember-appuniversum` to `3.4.1`

  * Update breaking changes
    * Use yielded `Group` from `AuToolbar` instead of `AuToolbar::Group`
    * Change `@skin='tertiary'` to `@skin='link'` on `AuButton`
    * Remove two-way binding on `AuInput`
  * Change how `appuniversum` SCSS is imported in the app.
  * Add a key to `AuDatePicker` localisation.

* Bump `@lblod/ember-rdfa-editor-lblod-plugins` to `17.0.0`
* Bump `@lblod/ember-environment-banner` to `0.5.0`


##### connected issues and PRs:

https://binnenland.atlassian.net/browse/GN-4801


### Setup

1. Checkout
2. `npm i` 

### How to test/reproduce

* Start RB
* Navigate around the app, make sure everything still works, try to input/change input of  input field and textareas that you see, see if there are errors in console.


### Checks PR readiness
